packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -147,6 +151,18 @@ class WebKitWebViewControllerCreationParams
   /// (Only available for iOS > 14.0)
   /// Defaults to false.
   final bool limitsNavigationsToAppBoundDomains;
+
+  /// Whether JavaScript can open windows without user interaction.
+  ///
+  /// Setting this to `true` allows JavaScript's `window.open()` to create
+  /// new windows automatically without requiring a user gesture.
+  ///
+  /// When `null`, the platform's native default is used:
+  /// - iOS: `false`
+  /// - macOS: `true`
+  ///
+  /// See https://developer.apple.com/documentation/webkit/wkpreferences/1536573-javascriptcanopenwindowsautomati
```

Comment:
nit, the link doesn't seem to be complete (although the website does seem to redirect you to the correct page). Also I doubt this will be rendered as a link in markdown (so people would have to copy and paste this into their browser's URL bar in order to open it).